ci-operator/config/stolostron/discovery/stolostron-discovery-backplane-2.0.yaml

```diff
@@ -0,0 +1,159 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+  stolostron_builder_go1.17-linux:
+    name: builder
+    namespace: stolostron
+    tag: go1.17-linux
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: stolostron
+    tag: go1.17-linux
+images:
+- dockerfile_path: Dockerfile.prow
+  from: base
+  inputs:
+    stolostron_builder_go1.17-linux:
+      as:
+      - registry.ci.openshift.org/stolostron/builder:go1.17-linux
+  to: discovery-operator
+- dockerfile_path: Dockerfile.test.prow
+  from: base
+  inputs:
+    stolostron_builder_go1.17-linux:
+      as:
+      - registry.ci.openshift.org/stolostron/builder:go1.17-linux
+  to: discovery-operator-tests
+- dockerfile_path: Dockerfile.testserver.prow
+  from: base
+  inputs:
+    stolostron_builder_go1.17-linux:
+      as:
+      - registry.ci.openshift.org/stolostron/builder:go1.17-linux
+  to: discovery-testserver
+promotion:
+  disabled: true
+  name: "2.5"
```

Comment:
Should this be "2.0"?

Reply:
Based on the example of the "work" directory. I think when we are still promoting to both release-2.5 and backplane-2.0, it should stay as "2.5". I think that will change once we remove it after the transition

Reply:
The promotion name/namespace is just to be able to keep different releases separate. It should stay `2.5` for now. I'll talk with David about what needs to happen after the MCE components are removed from ACM.